packages/devtools_app/test/primitives/utils_test.dart

```diff
@@ -337,6 +337,46 @@ void main() {
         expect((TimeRange()..end = Duration.zero).isWellFormed, isFalse);
         expect((TimeRange()..start = Duration.zero).isWellFormed, isFalse);
       });
+
+      test('offset', () {
```

Comment:
nit: this could be a group called `offset`, then it could be split into 4 individual unit tests

Reply:
done